playbooks/ci_latency_tests.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2023 Savoir-faire Linux, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
+- name: Launch sv timestamp logger on publisher and VM
+  hosts:
+    - publisher_machines
+    - VMs
+  tasks:
+    - name: Load Docker archive
+      command: "docker image load -i sv_timestamp_logger.tar"
+      become: yes
+
+    - name: Launch sv timestamp logger
```

Comment:
You will need to launch two different commands : 
- On publisher with hardware timestamping
- On vm without hw ts